### ISSUE TITLE
Hadoop Example

### ISSUE THREAD

**Issue by erictune** [kubernetes/kubernetes#2563]
A user requested an example of setting up a Hadoop cluster on Kubernetes.
https://groups.google.com/forum/#!topic/google-containers/cdeL_R2NYbY

Related to https://github.com/GoogleCloudPlatform/kubernetes/issues/1624

**Comment by bgrant0607** [comment]
We do plan to look at Hadoop, but not until after our 1.0 milestone. Please see our [roadmap](https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/roadmap.md) for our current focus.

**Comment by timothysc** [comment]
Closing this in favor of - https://github.com/kubernetes/application-images/pull/8

**Comment by elsonrodriguez** [comment]
I've started playing with deploying Hadoop on k8s, based on images from here: https://hub.docker.com/r/uhopper/hadoop/

Just about any variable for hdfs is tunable via environment variables via some clever bash.

WIP for k8s objects is here: https://bitbucket.org/erodriguez/hadoop-docker/src/b76b3ba132736acadb32273b7f617783c1208c13/kubernetes/?at=master
